src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+form {
+  margin-top: 20px;
+}
+
+.searchform {
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchform:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchformbig {
+  height: 70px;
+}
+
+.searchformsmall {
+  height: 42px;
+  /* width: 427px; */
+}
+
+.field {
+  width: 100%;
+  height: 100%;
+  border: none;
+  outline: none;
+  color: #3d4e61;
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
Add font-weight, it should be 300 by default, 400 for placeholder and 900 for focused style

Reply:
You now have only Avenir font-family